util/include/Handoff.hpp

```diff
@@ -60,10 +60,13 @@ class Handoff {
     {
       guard g(queue_lock_);
       task_queue_.push(std::move(f));
+      task_queue_size_ = task_queue_.size();
     }
     queue_cond_.notify_one();
   }
 
+  size_t size() { return task_queue_size_; }
```

Comment:
HI, I was aware that the value I  get is not the most accurate, and in the right time, I wanted to reduce the number of locks, and this value is used only for stats.
Anyway, rethinking since this is an API function and who knows who will use it in the future, I will do the change, the contention on this queue is not high, 1 consumer and 1 producer...
Thanks for the

Reply:
I don't understand thee idea of the change

Reply:
Just added an API to get the size of the handoff tasks queue..